app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/agenda_screen/components/AgendaCard.kt

```diff
@@ -173,6 +177,24 @@ fun AgendaCard(
                     .padding(end = DP.extraSmall)
                     .align(Alignment.End)
             )
+
+            // • Progress indicator for (for Events only)
+            toDateTime?.let {
+                AnimatedVisibility(zonedDateTimeNow.isAfter(fromDateTime) && zonedDateTimeNow.isBefore(toDateTime)) {
+                    Text(
+                        text = "In Progress: " +
+                                (100 - ( ( (toDateTime.toEpochSecond() - zonedDateTimeNow.toEpochSecond()).toFloat()
+                                  / (toDateTime.toEpochSecond() - fromDateTime.toEpochSecond()).toFloat() )*100))
+                                    .roundToInt() + "% Complete",
```

Comment:
Introducing some variables here would make this a lot more readable

Reply:
ok will do.